source/install/apt.rst

```diff
@@ -23,116 +23,89 @@ of the following distributions:
    :local:
 
 Package Contents
-================
-
-:``percona-server-mongodb-36``:
- Installs the ``mongo`` shell, import/export tools, other client utilities,
- server software, default configuration, and init.d scripts.
-
-:``percona-server-mongodb-36-server``:
- Contains the ``mongod`` server, default configuration files,
- and init.d scripts.
-
-:``percona-server-mongodb-36-shell``:
- Contains the ``mongo`` shell.
-
-:``percona-server-mongodb-36-mongos``:
- Contains the ``mongos`` sharded cluster query router.
-
-:``percona-server-mongodb-36-tools``:
- Contains Mongo tools for high-performance MongoDB fork from Percona.
-
-:``percona-server-mongodb-36-dbg``:
- Contains debug symbols for the server.
+================================================================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - Package
+     - Contains
+   * - percona-server-mongodb-40
+     - The ``mongo`` shell, import/export tools, other client
+       utilities, server software, default configuration, and init.d scripts.
+   * - percona-server-mongodb-40-server
+     - The :program:`mongod` server, default configuration files, and :dir:`init.d`
+       scripts
+   * - percona-server-mongodb-40-shell
+     - The ``mongo`` shell
+   * - percona-server-mongodb-40-mongos
+     - The ``mongos`` sharded cluster query router
+   * - percona-server-mongodb-40-tools
+     - Mongo tools for high-performance MongoDB fork from Percona
+   * - percona-server-mongodb-40-dbg
+     - Debug symbols for the server
 
 Installing from Percona Repositories
-====================================
+================================================================================
 
 It is recommended to install |PSMDB| from official Percona repositories:
 
-1. Configure Percona repositories as described in
-   `Percona Software Repositories Documentation
-   <https://www.percona.com/doc/percona-repo-config/index.html>`_.
-
-#. Install the required |PSMDB| package using :command:`apt-get`.
-   For example, to install the full package, run the following:
+|tip.run-all.root|
 
-   .. code-block:: bash
-
-      $ sudo apt-get install percona-server-mongodb-34
+1. Fetch the repository packages from Percona web: :bash:`wget https://repo.percona.com/apt/percona-release_0.1-8.$(lsb_release -sc)_all.deb`
```

Comment:
from what I see 0.1.9 was already release, but I would advise you to use the "latest" package in the docs because that way you won't need to update the docs on every release package update.
Example: `percona-release_latest.jessie_all.deb`
Check here: https://repo.percona.com/apt/